src/super_gradients/modules/detection_modules.py

```diff
@@ -76,13 +81,12 @@ def out_channels(self):
         return self._out_channels
 
     def forward(self, x):
-
+        y = x
```

Comment:
Don't think we really need this line right ?

Reply:
Indeed. Fixed in 652d3d267beecd50e214d60ad03c3d330baafd6c